store/sqlstore/emoji_store.go

```diff
@@ -63,6 +63,31 @@ func (es SqlEmojiStore) Save(emoji *model.Emoji) (*model.Emoji, *model.AppError)
 	return emoji, nil
 }
 
+// GetBy returns one active (not deleted) emoji, found by any one column (what/key).
+func (es SqlEmojiStore) GetBy(what string, key interface{}) (*model.Emoji, *model.AppError) {
```

Comment:
Can we make `GetBy` private? Ideally this would never be used by a client (i.e. without an appropriate index).